README.md

```diff
@@ -27,6 +27,15 @@ Use *globalize-webpack-plugin* if your application uses [Globalize][] for intern
 
 Starting from *v1.0.0*, only *webpack 2* is supported. If you need support for *webpack 1*, use our *v0.x* releases.
 
+##### Globalize vs. webpack vs. globalize-webpack-plugin versions
+
+| globalize     | webpack | globalize-webpack-plugin |
+| ------------- | ------- | ------------------------ |
+| ^1.1.0 <1.3.0 | ^1.9.0  | 0.3.x                    |
+| ^1.3.0        | ^1.9.0  | 0.4.x                    |
+| ^1.1.0 <1.3.0 | ^2.2.0  | 1.0.x                    |
+| ^1.3.0        | ^2.2.0  | 1.1.x                    |
+
```

Comment:
Any objection to releasing the support for globalize@^1.3.0 + webpack@^2.2.0 in GWP 1.1.x?